README.md

```diff
@@ -27,6 +28,10 @@ NOTE: If you are unable to purchase the Armchair Analysis data, [Ben Dilday](htt
 - pandas
 - scikit-learn
 
+## Node.js package requirements
```

Comment:
My thought is that parallels the list of python requirements above (which are listed in `requirements.txt` and installed on `pip install`) -- it wasn't clear to me from the code and the "This code currently requires Python 2.7" note above that `Node.js` was  required to be installed in order to run -- since Python uses `.json` also now and there are no `.js` files in the main directory.  But I could be persuaded.